src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/AuroraTopologyService.java

```diff
@@ -34,6 +34,8 @@
 import com.mysql.cj.conf.ConnectionUrl;
```

Comment:
General note. Namespace com.mysql.cj.jdbc.ha.* needs to be reviewed. Most of the classes there should be refactored wither to general plugin support, or to a particular plugin. In this case it should be failover plugin.

com.mysql.cj.jdbc.ha.ca.plugins.failover -> com.mysql.cj.jdbc.plugins.failover

Reply:
Was a bit hesitant to move the community code to a different folder in case of merge issues when we update the code